### PR TITLE
[FIX] Document PR title flow

### PR DESCRIPTION
**Pull request by NicolaVerbeeck**

### CODE REVIEW

**Review by srares76 — APPROVED** [review]

**Review by brapoprod — APPROVED** [review]

**Comment by brapoprod** [comment]
> LGTM, only remark is that now this behavior is not enforced. In the sense that I could choose to have a PR without any of the predefined tags ([FIX], etc). In this case it is automatically treated as a patch version. Which I believe is fine, but enforcing it could make people think about it more explicitly

can be an increment over time no?

File: images/pr_flow.svg
Comment (Matthiee) [inline images/pr_flow.svg:1]: So we can never have an NPM version with only the patch version being increased?

**Review by Matthiee — APPROVED** [review]

File: images/pr_flow.svg
Comment (NicolaVerbeeck) [reply images/pr_flow.svg:1]: @brapoprod did we miss a case? :)

**Review by Dvergar — APPROVED** [review]

File: images/pr_flow.svg
Comment (brapoprod) [reply images/pr_flow.svg:1]: yes we can

**Pull request merged**